scripts/docker/interactive/run-perfetto.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd "${PERFETTO_ROOT}" || exit
+
+tools/tmux -c celer.cfg -C out/linux -n
```

Comment:
Probably better to exec?
```suggestion
exec tools/tmux -c celer.cfg -C out/linux -n
```
Same comment about bash above.